### ISSUE TITLE
Sox::Error (no such file)

### ISSUE THREAD

**Issue by rosshatokay** [TrevorHinesley/carrierwave-audio#3]
Upon trying to process a conversion:

`include CarrierWave::Audio`
`process :convert => [{output_format: :mp3}]`

Rails logs this error

`Sox::Error (No such file or directory - sox --type mp3...`

How can I resolve this?

**Comment by TrevorHinesley** [comment]
That seems like Sox is not installed. Have you installed Sox on your OS?

**Comment by rosshatokay** [comment]
Hey, thanks for replying. I'll install it right now, but will this work in a production environment?

**Comment by TrevorHinesley** [comment]
Yes you'll need to install it on your production machine. If you're using Heroku, they don't allow you to install other packages I don't think.

**Comment by rosshatokay** [comment]
I'm assuming that I'd have to use a [custom heroku buildpack?](https://github.com/lepinsk/heroku-buildpack-soxl), 

if this is at all the optimal way of going through audio conversions, 
or are there any other libraries for that?

**Comment by TrevorHinesley** [comment]
Sorry for the late reply! Don't remember getting notified. Most likely you'll need a buildpack yes. And yes this is pretty standard, we've been using this in production for more than two years.
